app/javascript/active_admin/features/batch_actions.js

```diff
@@ -45,7 +46,7 @@ const toggleDropdown = function(condition) {
   }
 }
 
-const toggleAllChange = function(event) {
+const toggleAllChange = function(_event) {
```

Comment:
Not sure if these changes are ok or if these function are missing `event.preventDefault` maybe someone with a deeper knowledge can check faster than me